src/pages/home-page-slides/slide-add/slide-add.js

```diff
@@ -0,0 +1,32 @@
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { useStyles } from './slide-add.styles';
+import HomePageSlideForm from '../../../components/home-page-slide-form';
+import { getAvailableSlides} from '../../../redux/home-page-slides/home-page-slides.actions';
+import LoadingBar from '../../../components/loading-bar';
+
+const SlideAdd = () =>{
+  const styles = useStyles();
+  const dispatch = useDispatch();
+  const { loading, availableSlides } = useSelector(({ Slides }) => ({
+    loading: Slides.slideLoading,
+    availableSlides: Slides.availableSlides
+  }));
+  useEffect(() => {
+    dispatch(
+      getAvailableSlides()
+    );
+  }, [dispatch]);
+
+  const slideOrder = availableSlides.filter(slide=>slide.show===true).length + 1
```

Comment:
Add a proper aligning for the code, also you can simplify filter callback here:  `filter(slide => slide.show)`

Reply:
Fixed